app/javascript/spec/ansible-playbook-workflow/ansible-playbook-workflow.data.js

```diff
@@ -1,4 +1,7 @@
-export const ansiblePlaybookData = {
+/** Payload data received will be a string.
+ * eg: `{ "id": "1000", "payload": { "name": 'John Doe', "age": '30', "email": "johndoe@example.com", ... } }`,
+*/
+export const ansiblePlaybookData = JSON.stringify({
```

Comment:
Is `JSON.stringify` correct if the payload type is yaml?  Tecnically JSON is a subset of YAML, so this might be ok?

Reply:
Using `JSON.stringify` would print the code, but It would look like this...

<img width="764" alt="Screenshot 2023-10-25 at 9 45 29 AM" src="https://github.com/ManageIQ/manageiq-ui-classic/assets/87487049/723e40e2-bdb4-48e3-af09-981d9df5c1a9">

Now I have used the YAML payload in the spec file (from one of the existing records)

Reply:
Hey @Fryguy , is this change ok?

Reply:
It just feels like a bad test because it doesn't test an expected playbook payload.  Let me see if I can give an example

Reply:
Oh wait, I see you put a "real" payload below - think you can remove the JSON.stringify now.